dev/plugins/hu.elte.txtuml.api.stdlib/src/hu/elte/txtuml/api/stdlib/timers/Timer.xtxtuml

```diff
@@ -28,18 +35,38 @@
 	 *            the signal which is to be sent after the delay
 	 * @param millisecs
 	 *            the time in millisecs to wait before sending the signal
-	 * @return a timer instance to manage this delayed send operation before it
+	 * @return a handle object to manage this delayed send operation before it
 	 *         happens
 	 */
-	static Timer start(ModelClass targetObj, Signal signal, int millisecs) {
-		return new TimerImpl(targetObj, signal, millisecs);
+	public static external-body Timer start(Signal sig, ModelClass targetObj, int millisecs) {
```

Comment:
Results of an internal discussion (@kovacsgabor):
* It is OK that the `Timer(Signal sig, ModelClass targetObj, int millisecs)` is external, because we do not allow `Signal` and `ModelClass` as parameter types. (TODO: Check if this could be allowed later: Does UML standard allow this? Is it hard to implement in C++? @nboldi, @nagyandras95)
* We expect the user to know that an `Action.create` calls a constructor, and, if the constructor is external, then the create action can only be called in external or external-body context.